### DISCUSSION TITLE
No recommended template anymore?

### DISCUSSION THREAD

**Discussion by Mishasama** [community/community#10236] [Other Feature Feedback, Questions, & Ideas]
Before the public beta, it will display three default templates for users to choose from.
Why remove it?

**Comment by yanndinendal** [comment]
I don't know if it's because the suggestions were removed or maybe they only appear before you start using them.

So you see 3 great lists suggestions, think: cool, let's start using them; click "future ideas"... Boom! the other 2 disappear; it now only shows **your** list. Sadness ensues. :smiling_face_with_tear:

**Reply by navinthomas** [reply]
Same issue here. I thought there were already created for me. I created a new one for a specific case, and the other 3 ones disappeared. That's both poor and confusing behaviour.
